blockFromPersonalCalendar.gs

```diff
@@ -2,6 +2,11 @@
  * This script takes events from a calendar (pressumably personal), and blocks the times
  * in which there are events in another one (pressumably professional)
  * 
+ * Adapated from https://github.com/marianosimone/google-app-scripts/blob/main/blockFromPersonalCalendar.gs.
+ * My changes:
+ * - handle multiple secondary calendars
+ * - specify a color for the created events
+ * 
```

Comment:
would you mind removing this and changing the first line of my comment instead and say "This script takes events from a list of calendars (presumably...)"